src/Clapi/TextSerialisation.hs

```diff
@@ -0,0 +1,33 @@
+module Clapi.TextSerialisation where
+import Data.Monoid
+import qualified Data.Map as Map
+import Blaze.ByteString.Builder (Builder)
+import Blaze.ByteString.Builder.Char.Utf8 (fromString, fromChar)
+import qualified Clapi.Serialisation as Wire
+import Clapi.Types (Time(..), ClapiValue(..))
+
+typeTag :: ClapiValue -> Char
+typeTag (ClBool _) = 'B'
+typeTag v = Wire.typeTag v
+
+fss :: Show a => a -> Builder
+fss a = fromString $ show a
```

Comment:
Not that it's worth it at all, but this is a grand one to write as point-free if you fancy that sort of thing :)
```haskell
fss = fromString . show
```